draft-ietf-tls-tls13.md

```diff
@@ -4609,6 +4653,9 @@ Old versions of TLS permitted the use of very low strength ciphers.
 Ciphers with a strength less than 112 bits MUST NOT be offered or
 negotiated for any version of TLS for any reason.
 
+The security of SSL 3.0 {{SSL3}} is considered insufficient for the reasons enumerated
+in {{RFC7568}}, and MUST NOT be negotiated for any reason.
+
```

Comment:
nit: SSL2 should be before SSL3; if you're moving this around, just stick it after the SSL2 bit below.

Alternatively, we could just merge them to simplify things a little bit:
"The security of both SSL 2.0 {{SSL2}} and SSL 3.0 {{SSL3}} are considered insufficient for the reasons enumerated in {{RFC6176}} and {{RFC7568}}, and MUST NOT be negotiated for any reason."